src/misc/like.ts

```diff
@@ -0,0 +1,243 @@
+import { DatabaseEntry } from '../common/datas/database';
+import { dataType, dataHolder } from '../common/datas/holder';
+import { dataView } from '../common/datas/view';
+import * as yup from 'yup';
+import { Bukkit, OfflinePlayer } from 'org.bukkit';
+import { UUID } from 'java.util';
+import { isAdminAccount } from '../common/helpers/player';
+import { Player } from 'org.bukkit.entity';
+import { CommandSender } from 'org.bukkit.command';
+
+const playerLikesDataType = dataType('playerLikesData', {
+  playerUuidList: yup.array().of(yup.string().required()),
+});
+
+const playerLikesType = dataType('playerLikes', {
+  count: yup.number(),
+  lastAutoRemoveDate: yup.number(),
+  cooldownEnds: yup.number(),
+});
+
+const view = dataView(
```

Comment:
```suggestion
const topPlayers = dataView(
```
Or something else, but *not* just 'view'. Took me a while to figure this out while reading the functions below.